resources-domain/resources-service/migrations/20230809143655-create-ResourcesAudits.js

```diff
@@ -0,0 +1,203 @@
+        "user" text NOT NULL,
+        "tableName" text NOT NULL,
+        operation text NOT NULL,
+        "oldRecord" jsonb,
+        "newRecord" jsonb,
+        timestamp_trx timestamp with time zone NOT NULL,
+        timestamp_stm timestamp with time zone NOT NULL,
+        timestamp_clock timestamp with time zone NOT NULL,
+        PRIMARY KEY (id)
+        );`,
+    );
+    console.log('Table "Audits" created');
+
+    await queryInterface.sequelize.query(`
+      CREATE OR REPLACE FUNCTION resources.audit_trigger()
+        RETURNS trigger
+        LANGUAGE 'plpgsql'
+        NOT LEAKPROOF
+      AS $BODY$
+      DECLARE
+        resource_audit_row resources."Audits";
+      BEGIN
+        IF TG_WHEN <> 'AFTER' THEN
+          RAISE EXCEPTION 'audit_trigger() may only run as an AFTER trigger';
+        END IF;
+
+        IF (TG_LEVEL <> 'ROW' OR (TG_OP <> 'UPDATE' AND TG_OP <> 'INSERT' AND TG_OP <> 'DELETE')) THEN
+          RAISE EXCEPTION 'audit_trigger() added as trigger for unhandled case: %, %',TG_OP, TG_LEVEL;
+          RETURN NULL;
+        END IF;
+        
+        resource_audit_row = ROW(
+          nextval('"Audits_id_seq"'::regclass), -- new audit id
+          current_user,                         -- the current DB user
+          TG_TABLE_NAME,                        -- target tabla name
+          TG_OP,                                -- operation performed on target row
+          to_jsonb(OLD),                        -- target record previous state
+          to_jsonb(NEW),                        -- target record new state
+          current_timestamp,                    -- transaction timestamp
+          statement_timestamp(),                -- statement timestamp
+          clock_timestamp()                     -- Current date and time (changes during statement execution)
+        );
+
+        INSERT INTO resources."Audits" VALUES (resource_audit_row.*);
+        RETURN NULL;
+      END
+      $BODY$;
+    `);
+    console.log('Function "audit_trigger" created.');
+
+    await queryInterface.sequelize.query(
+      `ALTER FUNCTION resources.audit_trigger() OWNER TO resources`,
+    );
+    console.log('Function "audit_trigger" ownership altered.');
+
+    await queryInterface.sequelize.query(`
+      CREATE TRIGGER "Resources_audit_trigger"
+      AFTER INSERT OR DELETE OR UPDATE 
+      ON resources."Resources"
+      FOR EACH ROW
+      EXECUTE FUNCTION resources.audit_trigger();
+    `);
+    console.log('Trigger Resources_audit_trigger created');
+
+    await queryInterface.sequelize.query(`
+      CREATE TRIGGER "ResourceStringAttributes_audit_trigger"
+      AFTER INSERT OR DELETE OR UPDATE 
+      ON resources."ResourceStringAttributes"
+      FOR EACH ROW
+      EXECUTE FUNCTION resources.audit_trigger();
+    `);
+    console.log('Trigger ResourceStringAttributes_audit_trigger created');
+
+    await queryInterface.sequelize.query(`
+      CREATE TRIGGER "Globals_audit_trigger"
```

Comment:
We don't want a trigger on Globals - that table is deleted from the schema - I think it's still hanging around on dev for some reason but it's gone from everywhere else, we should just manually delete it from dev

This is why the service tests are failing for this PR

```
Migration failed. Retrying... MigrationError: Migration 20230809143655-create-ResourcesAudits.js (up) failed: Original error: relation "resources.Globals" does not exist
```